tests/data/AnsysSound_Orders_MultipleParameters dBSPL - InvertedContols_2024R2_20241205.txt

```diff
@@ -0,0 +1,24 @@
+AnsysSound_Orders_MultipleParameters 1
+dBSPL
+charge
+%
+RPM
+tr/min
```

Comment:
is this total revolutions/min? I'm not familiar with the unit.

Reply:
@mattbennett72 I hadn't realized that until you asked, but this is French! (I took the file as it was given to me ^^) 'tr' is short for 'tour' which means revolution indeed. "tour par minute" is a common French way to refer to rpm. I'll will change this here